tools/halide_image_info.h

```diff
@@ -0,0 +1,333 @@
+// This header defines a several methods useful for debugging programs that
```

Comment:
I'd like it if this change was made in a separate branch from master, not the hexagon branch.